src/liquidity-positions/events.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * LP specific events.
+ */
+export enum LiquidityEventName {
+  ADD_LIQUIDITY = 'Add Liquidity',
+  COLLECT_LIQUIDITY = 'Collect Liquidity',
+  MIGRATE_LIQUIDITY = 'Migrate Liquidity',
+  REMOVE_LIQUIDITY = 'Remove Liquidity',
```

Comment:
Let's add `Submitted` onto these to clarify these are not confirmed to be successful on chain